{"ERROR": {"1520307098": {"duration": 0.02089834213256836, "exception": {"message": "Invalid template library specified. ImportError raised when trying to load 'my_web.apps.article.templatetags.custom_markdown': No module named 'markdown2'", "raw": "Invalid template library specified. ImportError raised when trying to load 'my_web.apps.article.templatetags.custom_markdown': No module named 'markdown2'", "traceback": [{"file": "/usr/local/lib/python3.6/site-packages/django/core/handlers/base.py", "line": 126, "method": "_get_response"}, {"file": "/django_env/my_web/apps/article/views.py", "line": 35, "method": "articleListView"}, {"file": "/usr/local/lib/python3.6/site-packages/django/shortcuts.py", "line": 36, "method": "render"}, {"file": "/usr/local/lib/python3.6/site-packages/django/template/loader.py", "line": 61, "method": "render_to_string"}, {"file": "/usr/local/lib/python3.6/site-packages/django/template/loader.py", "line": 12, "method": "get_template"}, {"file": "/usr/local/lib/python3.6/site-packages/django/template/loader.py", "line": 66, "method": "_engine_list"}, {"file": "/usr/local/lib/python3.6/site-packages/django/template/utils.py", "line": 88, "method": "all"}, {"file": "/usr/local/lib/python3.6/site-packages/django/template/utils.py", "line": 88, "method": "<listcomp>"}, {"file": "/usr/local/lib/python3.6/site-packages/django/template/utils.py", "line": 79, "method": "__getitem__"}, {"file": "/usr/local/lib/python3.6/site-packages/django/template/backends/django.py", "line": 25, "method": "__init__"}, {"file": "/usr/local/lib/python3.6/site-packages/django/template/backends/django.py", "line": 43, "method": "get_templatetag_libraries"}, {"file": "/usr/local/lib/python3.6/site-packages/django/template/backends/django.py", "line": 108, "method": "get_installed_libraries"}, {"file": "/usr/local/lib/python3.6/site-packages/django/template/backends/django.py", "line": 125, "method": "get_package_libraries"}], "type": "django.template.library.InvalidTemplateLibrary"}, "raw": "{'request': {'method': 'GET', 'meta': {'http_host': 'www.spreader.online', 'http_x_real_ip': '218.76.54.47', 'http_x_forwarded_for': '218.76.54.47', 'http_user_agent': 'Mozilla/5.0 (Macintosh; Intel Mac OS X 10_12_6) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/64.0.3282.186 Safari/537.36', 'remote_addr': '172.21.0.4', 'path_info': '/home/'}, 'path': '/home/', 'scheme': 'http', 'data': {}, 'user': 'wim'}, 'exception': {'message': \"Invalid template library specified. ImportError raised when trying to load 'my_web.apps.article.templatetags.custom_markdown': No module named 'markdown2'\", 'type': 'django.template.library.InvalidTemplateLibrary', 'traceback': [{'file': '/usr/local/lib/python3.6/site-packages/django/core/handlers/base.py', 'line': 126, 'method': '_get_response'}, {'file': '/django_env/my_web/apps/article/views.py', 'line': 35, 'method': 'articleListView'}, {'file': '/usr/local/lib/python3.6/site-packages/django/shortcuts.py', 'line': 36, 'method': 'render'}, {'file': '/usr/local/lib/python3.6/site-packages/django/template/loader.py', 'line': 61, 'method': 'render_to_string'}, {'file': '/usr/local/lib/python3.6/site-packages/django/template/loader.py', 'line': 12, 'method': 'get_template'}, {'file': '/usr/local/lib/python3.6/site-packages/django/template/loader.py', 'line': 66, 'method': '_engine_list'}, {'file': '/usr/local/lib/python3.6/site-packages/django/template/utils.py', 'line': 88, 'method': 'all'}, {'file': '/usr/local/lib/python3.6/site-packages/django/template/utils.py', 'line': 88, 'method': '<listcomp>'}, {'file': '/usr/local/lib/python3.6/site-packages/django/template/utils.py', 'line': 79, 'method': '__getitem__'}, {'file': '/usr/local/lib/python3.6/site-packages/django/template/backends/django.py', 'line': 25, 'method': '__init__'}, {'file': '/usr/local/lib/python3.6/site-packages/django/template/backends/django.py', 'line': 43, 'method': 'get_templatetag_libraries'}, {'file': '/usr/local/lib/python3.6/site-packages/django/template/backends/django.py', 'line': 108, 'method': 'get_installed_libraries'}, {'file': '/usr/local/lib/python3.6/site-packages/django/template/backends/django.py', 'line': 125, 'method': 'get_package_libraries'}], 'raw': \"Invalid template library specified. ImportError raised when trying to load 'my_web.apps.article.templatetags.custom_markdown': No module named 'markdown2'\"}, 'duration': 0.02089834213256836}", "request": {"data": {}, "meta": {"http_host": "www.spreader.online", "http_user_agent": "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_12_6) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/64.0.3282.186 Safari/537.36", "http_x_forwarded_for": "218.76.54.47", "http_x_real_ip": "218.76.54.47", "path_info": "/home/", "remote_addr": "172.21.0.4"}, "method": "GET", "path": "/home/", "scheme": "http", "user": "wim"}}}}
